package.json

```diff
@@ -1,60 +1,61 @@
 {
-  "name": "bam.iobio",
-  "version": "1.0.0",
-  "scripts": {
-    "serve": "DEBUG=http nodemon --inspect=127.0.0.1:3001  --ignore client/ www.js",
-    "build": "webpack --mode production",
-    "build-docker": "BUILD_ENV_LOCAL_BACKEND=true webpack --mode production",
-    "watch": "webpack --mode development --watch",
-    "test": "mocha-webpack --webpack-config webpack.test.config.js --require client/spec/spec_helper.js \"client/spec/**/*_spec.js\""
-  },
-  "dependencies": {
-    "bootstrap": "^3.4.1",
-    "connect-history-api-fallback": "^1.6.0",
-    "d3": "^3.5.17",
-    "debug": "^2.6.9",
-    "express": "^4.16.4",
-    "fibridge-host": "^0.16.0",
-    "iobio-api-client": "^0.10.0",
-    "iobio-launch": "^0.7.0",
-    "jquery": "^3.3.1",
-    "js-cookie": "^2.2.0",
-    "less-loader": "^4.1.0",
-    "qs": "^6.7.0",
-    "serve-favicon": "^2.5.0",
-    "v-tooltip": "^2.0.0-rc.33",
-    "vue": "^2.6.10",
-    "vue-analytics": "^5.16.4",
-    "vue-router": "^3.0.5"
-  },
-  "devDependencies": {
-    "babel-core": "^6.26.3",
-    "babel-loader": "^7.1.5",
-    "babel-preset-latest": "^6.24.1",
-    "bluebird": "^3.5.4",
-    "chai": "^4.2.0",
-    "chai-almost": "^1.0.1",
-    "clean-webpack-plugin": "^0.1.19",
-    "cross-env": "^5.2.0",
-    "css-loader": "^0.28.11",
-    "extract-text-webpack-plugin": "^2.1.2",
-    "file-loader": "^3.0.1",
-    "jsdom": "^11.12.0",
-    "jsdom-global": "^3.0.2",
-    "mocha": "^3.4.2",
-    "mocha-webpack": "^0.7.0",
-    "node-sass": "^4.11.0",
-    "nodemon": "^1.18.10",
-    "raw-loader": "^0.5.1",
-    "sass-loader": "^6.0.7",
-    "sinon": "^2.3.5",
-    "sinon-chai": "^2.11.0",
-    "style-loader": "^0.18.2",
-    "url-loader": "^1.1.2",
-    "vue-loader": "^15.7.0",
-    "vue-template-compiler": "^2.6.10",
-    "vue-test-utils": "^1.0.0-beta.11",
-    "webpack": "^4.29.6",
-    "webpack-cli": "^3.3.0"
-  }
+	"name": "bam.iobio",
+	"version": "1.0.0",
+	"type": "module",
```

Comment:
this is a superfluous new attribute in the manifest file.
is it required for Prettier? otherwise, let's include it in a separate PR where it is relevant.

Reply:
```suggestion
```

Reply:
This is required to use export default with prettier.config.js, though I can probably use module.exports instead and remove this
https://prettier.io/docs/en/configuration